### PR TITLE
Update release process

### PR DESCRIPTION
**Pull request by bukzor**

### CODE REVIEW

File: bin/release [hunk 1]
```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+set -e
+source ./venv/bin/activate
+
+set -ux
+rm -r ./dist/
+git fetch yelp
```
Comment (asottile) [inline bin/release]: mine checkout isn't set up like this, you're making assumptions

File: bin/release [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+git checkout yelp/stable -B stable
+git tag -f v"$(python venv_update.py --version)"
+git push yelp --tags
+python setup.py sdist bdist_wheel
+python setup.py register
```
Comment (asottile) [inline bin/release]: this only needs to happen once

Comment (bukzor) [inline bin/release]: It's idempotent and quick.
It covers the "if you need to set up pypi auth" of the old `.md`.

File: bin/release (hunk 1 above)
Comment (bukzor) [inline bin/release]: That's not different from the LHS, and I don't see a way to do without assumptions.

File: bin/release (hunk 2 above)
Comment (asottile) [inline bin/release]: not quick for me, I've chosen to type my password in instead of having it on disk

File: bin/release (hunk 1 above)
Comment (asottile) [inline bin/release]: seems like a script that's useful to you but not anybody else

**Pull request closed without merging**